themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties

```diff
@@ -0,0 +1,314 @@
+loginTimeout=Zbyt dłużo czasu zajeło logowanie. Proces logowania rozpocnie się od nowa.
+oauthGrantTitle=Przydziel dostęp dla {0}
+oauthGrantTitleHtml={0}
+errorTitle=Przykro nam...
+errorTitleHtml=<strong>Przykro</strong> nam...
+emailVerifyTitle=Weryfikacja e-maila
+emailForgotTitle=Nie pamiętasz hasła?
+updatePasswordTitle=Aktualizacja hasła
+codeSuccessTitle=Kod sukcesu
+codeErrorTitle=Kod błędu\: {0}
+displayUnsupported=Żądany typ wyświetlania jest nieobsługiwany
+browserRequired=Do zalogowania wymagana jest przeglądarka
+browserContinue=Przeglądarka jest wymagana by dokończyć logowanie
+browserContinuePrompt=Otworzyć przeglądarkę i kontynuować logowanie? [t/n]\:
+browserContinueAnswer=t
+
+
+termsTitle=Regulamin
+termsText=<p>Regulamin, który należy zdefiniować</p>
+termsPlainText=Regulamin, który należy zdefiniować.
+
+recaptchaFailed=Błędna Recaptcha
+recaptchaNotConfigured=Recaptcha jest wymagana, ale nie skonfigurowana
+consentDenied=Zgoda odrzucona.
+
+noAccount=Nie masz konta?
+username=Nazwa użytkownika (login)
+usernameOrEmail=Nazwa użytkownika lub e-mail (login)
+firstName=Imię
+givenName=Nadane imię
+fullName=Pełne imię i nazwisko
+lastName=Nazwisko
+familyName=Nazwisko rodowe
+email=E-mail
+password=Hasło
+passwordConfirm=Potwierdź hasło
+passwordNew=Nowe hasło
+passwordNewConfirm=Potwierdzenie nowego hasła
+rememberMe=Zapamiętaj mnie
+authenticatorCode=Kod jednorazowy
+address=Adres
+street=Ulica
+locality=Miejscowość
+region=Województwo
+postal_code=Kod pocztowy
+country=Państwo
+emailVerified=Email zweryfikowany
+gssDelegationCredential=Świadectwo przekazania uprawnień GSS
+
+profileScopeConsentText=Profil użytkownika
+emailScopeConsentText=Adres email
+addressScopeConsentText=Adres
+phoneScopeConsentText=Numer telefonu
+offlineAccessScopeConsentText=Dostęp offline
+samlRoleListScopeConsentText=Moje role
+
+loginTotpIntro=Aby uzyskać dostęp do tego konta, musisz skonfigurować generator haseł jednorazowych
+loginTotpStep1=Zainstaluj jedną z następujących aplikacji na telefonie komórkowym
+loginTotpStep2=Otwórz aplikację i zeskanuj kod kreskowy
+loginTotpStep3=Wprowadź jednorazowy kod podany przez aplikację i kliknij Prześlij aby zakończyć konfigurację
+loginTotpManualStep2=Otwórz aplikację i wprowadź klucz
+loginTotpManualStep3=Użyj poniższych wartości konfiguracji, jeśli aplikacja pozwala na ich ustawienie
+loginTotpUnableToScan=Nie można skanować?
+loginTotpScanBarcode=Zeskanować kod paskowy?
+loginTotpOneTime=Kod jednorazowy
+loginTotpType=Typ
+loginTotpAlgorithm=Algorytm
+loginTotpDigits=Cyfry
+loginTotpInterval=Interwał
+loginTotpCounter=Licznik
+
+loginTotp.totp=Oparte o czas
+loginTotp.hotp=Oparte o licznik
+
+
+oauthGrantRequest=Czy przyznajesz te uprawnienia dostępu?
+inResource=w
+
+emailVerifyInstruction1=Została wysłana do Ciebie wiadomość e-mail z instrukcjami jak zweryfikować swój adres e-mail.
+emailVerifyInstruction2=Nie otrzymałem kod weryfikacyjny w wiadomości e-mail?
```

Comment:
This sentence doesn't look right.. maybe `Nie otrzymałem kodu weryfikacyjnego w wiadomości e-mail?`

Reply:
done, thanks